### DISCUSSION TITLE
Ecwolf does not open and does not run

### DISCUSSION THREAD

**Discussion by ghost** [eduardofilo/RG350_adam_image#78] [Ports]
I have noticed that Ecwolf does not load. Just a black screen before getting back to the menu.

Logs states that . ecwolf.opk does not find the libbz2.so.1.0 library.

Any clue why this library is missing?

**Comment by eduardofilo** [comment]
Well, surely that OPK is compiled againts the toolchain for old OpenDingux. Someone should adapt the project to run over OpenDingux beta. It seems that the project is this: https://github.com/JohnnyonFlame/ecwolf

**Comment by ghost** [comment]
The simple adaptation is to statically link the libbz2.so.1.0.8 (latest version) the point is that it is not the opk the problem but the missing linked library in /usr/lib/ folder of OD beta. All programmers knows that when it is possible to use linked library it is always good to save space for just a few compromise in performance. The project of ecwolf does not need otherwise any adaptation.

**Comment by a-dekker** [comment]
Here is a newly compiled OD beta version that doesn't even use the shared libbz2 object https://onedrive.live.com/?authkey=%21AAJGVs83oikU%2DCU&id=6D38D58310548416%21930&cid=6D38D58310548416
